cgs_vmc/layers.py

```diff
@@ -405,3 +406,38 @@ def _build(self, inputs: tf.Tensor) -> tf.Tensor:
 
     batch_components = [self._mps_var[:, :, index] for index in index_inputs]
     return tf.stack(batch_components, axis=0)
+
+
+class GraphConvLayer(snt.AbstractModule):
+  """GraphConvLayer module with adjacency list"""
+
+  def __init__(
+      self,
+      output_channels: int,
+      adj: np.ndarray,
+      name: str = 'gnn_layer'):
+    """Constructs GraphConvLayer moduel.
+
+    Args:
+      output_channels: Number of output channels in convolution.
+      adj: Adjacency list of the graph that stores indices of neighbors and itself for every site, with shape [n_site, num_neighbor].
```

Comment:
Maximum line length by many standards is 80 characters. (See Conv2dPeriodic for an example of breaking doc-string into multiple lines)